MastodonSDK/Sources/MastodonSDK/Entity/Mastodon+Entity+Account.swift

```diff
@@ -84,6 +84,64 @@ extension Mastodon.Entity {
     }
 }
 
+extension Mastodon.Entity.Account: Hashable {
```

Comment:
Apparently there is already a conformance of this typ to `Hashable` and it's only taking into account the `id` ...
 <img width="1013" alt="Bildschirmfoto 2023-11-14 um 11 53 07" src="https://github.com/mastodon/mastodon-ios/assets/126418/16570183-a868-49fe-ac06-09244f011517">

Reply:
Oh, I didn't see that 🙈